app/src/main/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -135,25 +136,37 @@ class MessagingService : FirebaseMessagingService() {
      *
      */
     private suspend fun sendNotification(data: Map<String, String>) {
-        var group = data["group"]
-        if (!group.isNullOrBlank()) group = GROUP_PREFIX + group
+        val notificationManager =
+            getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
```

Comment:
Why are we moving away from `NotificationManagerCompat`?

Reply:
Because i needed the extension methods of the notification manager. It was easier (for me at least) to use again the notification manager. I'll try to switch back again to the compat one.

Reply:
I just worry about older devices, the compat layer adds some safety for us.